src/server_mocker_instruction.rs

```diff
@@ -7,7 +7,7 @@ pub type BinaryMessage = Vec<u8>;
 
 /// Type of network instruction executed by the server mocker.
 #[derive(Debug, Clone, PartialEq)]
-pub enum ServerMockerInstruction {
+pub enum Instruction {
```

Comment:
Not sure of that tbh... Don't you think it will act like tech debt in case the crate is extended in the future?